README.md

```diff
@@ -3,7 +3,7 @@ The official COVID-19 app of the University of Illinois. Powered by the [Rokwire
 
 ## Requirements
 
-### [Flutter](https://flutter.dev/docs/get-started/install) v2.0.6
+### [Flutter](https://flutter.dev/docs/get-started/install) v2.0.2
```

Comment:
```suggestion
### [Flutter](https://flutter.dev/docs/get-started/install) v2.2.2
```

Reply:
Going by the CHANGELOG entry, this looks like 2.2.2. I'm committing this. Please fix this later if this is not correct.

Reply:
I am sorry about this, @sandeep-ps. It was a typo:
https://github.com/rokwire/safer-illinois-app/commit/ae7bc787563cca37066fb6506fc920ffb8ccd675